compiler/util-klib-metadata/src/org/jetbrains/kotlin/library/metadata/KlibMetadataClassDataFinder.kt

```diff
@@ -22,12 +23,13 @@ class KlibMetadataClassDataFinder(
 ) : ClassDataFinder {
     val nameList = fragment.getExtension(KlibMetadataProtoBuf.className).orEmpty()
 
+    val classIdToIndex: Map<ClassId, Int> = HashMap<ClassId, Int>().apply {
```

Comment:
It can be written simpler:
```
    val classIdToIndex: Map<ClassId, Int> = buildMap {
        nameList.forEachIndexed { index, value -> this[nameResolver.getClassId(value)] = index }
    }
```